contracts/blade/ChildERC20Predicate.sol

```diff
@@ -171,12 +190,25 @@ contract ChildERC20Predicate is IChildERC20Predicate, Predicate, Initializable,
         emit ERC20Withdraw(rootToken, address(childToken), msg.sender, receiver, amount);
     }
 
+    function _withdrawRollback(bytes calldata data) private {
+        (address depositToken, address depositor, , uint256 amount) = abi.decode(
```

Comment:
Same as in `ChildERC1155Predicate`. Decode the `sender` and `receiver` addresses, and use them accordingly.